### ISSUE TITLE
update script caused usb hub not to function 

### ISSUE THREAD

**Issue by jdalmanza** [MiSTer-devel/Updater_script_MiSTer#79]
ran the update script today (twice, form 2 different installs) and after the reboot the usb hub that has worked for months now no longer functions

I have been using the following usb hub since I first started using MiSTer several months ago:
https://www.amazon.com/LoveRPi-MicroUSB-Port-Black-Raspberry/dp/B01HYJLZH6/ref=sr_1_3?crid=22VTQQH42DHCA&keywords=otg+usb+hub&qid=1636765273&sprefix=otg+usb+h%2Caps%2C161&sr=8-3

... and the hub has worked just fine for a corded keyboard and iBuffalo snes style gamepad 

Today I ran the updater from 'update_all (v1.4)' and once the update was complete, after the reboot I got no response from keyboard or gamepad 

I then flashed a new sd card (using Mr. Fusion 2.3) , in the same DE10nano, kept the same hub plugged in and was able to initiate the included update script and again once the update was completed there was no response from wither input device

Seems like there is some update that dropped the support for hub or something?

Please let me know what kind of logging you need, it is very simple to reproduce on my end, but unfortunately, unless you can grab logs off of the sd card, i am not able to enter any interactive commands in the failed state

**Comment by sorgelig** [comment]
nothing is dropped. Everything supposed to be supported is still supporting. 
I don't know what chip is inside this hub particularly, but i have same looking hub (but white) using in my NESPi 4 case adoption, and it works ok. Chip inside my hub is FE1.1S.
Does it start to work if you turn off the MiSTer and then turn on? Does it start to work after reboot of MiSTer?
Btw, issue has nothing to do with updater script.

**Comment by jdalmanza** [comment]
I tried, unplugging and plugging back in, I tried warm and cold boots, no response from any device. The hub does work, I can flash a new card from Mr. Fusion and the devices plugged into it just fine, it's only after updating that they no longer respond 

After a few searches, it seems that the likely issue is the Linux upgrade so if need be I can close this ticket and open one in the appropriate spot. 

The only thing I have not tried so far is creating a copy of the old Linux folder and replacing it after the update is complete but I can try that later

**Comment by sorgelig** [comment]
it's unclear which to which release of Linux you compare. Some USB compatibility happened after migrating from Kernel 4.x to 5.x.. So it's unclear if problem appeared just in this new build or you simply didn't update since 4.x till now.

**Comment by jdalmanza** [comment]
Not sure when the kernel was updated, but I had a current build up to about a month or so ago, but then I ran the update script and the problem happened. Then as a troubleshooting step, to rule out that the hub went bad spontaneously, I did a fresh flash from the latest mr fusion and updated from that

**Comment by jdalmanza** [comment]
After some troubleshooting, the most recent "fresh" flash upgrades Linux from 210917 -> 211112

Unfortunately I cannot tell which version I was running before this most recent upgrade that broke it. Is there a way to get the version prior to 211112 and test with that?

Also, shod I open a ticket specific to the Linux repo?
